src/test/kotlin/com/atlassian/performance/tools/report/jstat/JstatGcutilParserTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.atlassian.performance.tools.report.jstat
+
+import org.assertj.core.api.Assertions
+import org.junit.Test
+
+class JstatGcutilParserTest {
+    private val system = "test"
+
+    @Test
+    fun shouldParseJstat() {
+        val jstatCsv = this::class.java.getResourceAsStream("./jstat.csv")
+
+        val metrics = JstatGcutilParser().parse(jstatCsv, system)
+
+        Assertions.assertThat(metrics.map { it.toString() })
+            .contains(
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_SURVI_0, value=0.0, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_SURVI_1, value=0.0, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_EDEN, value=42.01, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_OLD, value=0.0, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_META, value=17.29, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_COMPRESSED_CLASS, value=19.76, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_YOUNG_GEN_GC, value=0.0, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_YOUNG_GEN_GC_TIME, value=0.0, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_FULL_GC, value=0.0, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_FULL_GC_TIME, value=0.0, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:41Z, dimension=JSTAT_TOTAL_GC_TIME, value=0.0, system='test')",
+                "SystemMetric(start=2018-01-18T13:58:43Z, dimension=JSTAT_SURVI_0, value=0.0, system='test')"
+            )
+    }
+
+    @Test
+    fun shouldNotRelyOnColumnsOrder() {
```

Comment:
Can we split the commit into a red-green pair?

Reply:
Short answer: No
Longer answer: Before the change, we didn't use headers in the CSV file, so there were only numbers + contract on the order of the column. Shuffling CSV without a header makes no sense. Switching to reading the header from a file will automatically give us the ability to shuffle. The test is to make it explicit we stop to rely on the order, not to make it a side change.

Reply:
> Shuffling CSV without a header makes no sense.

Some users are impacted by JPERF-462. Do they shuffle? Or do they use different JDKs, resulting in different (but still deterministic) column order? If it's the latter, we could add a `adopt-jstat.csv` and perform the same unit test.

The fix requires adding a header row, ok, but that'd be the part of the green commit.

Reply:
Yes. But I don't have an adopt-jstat.log to reproduce the issue.

Reply:
The reporter of https://ecosystem.atlassian.net/browse/JPERF-462 should have it ;)

Reply:
Thanks for the red-green for JPERF-464 👍 
Is it feasible to obtain that AdoptJDK `jstat.log` and go red-first for JPERF-462?

Reply:
I would go for it, but I'm in a deep sheet with my sprint's goal and the PR is a side improvement to the CI I work on in the meantime.